core/src/Http/RedirectUriHelper.cs

```diff
@@ -44,6 +53,19 @@ public static void Validate(Uri redirectUri)
                     CoreErrorMessages.RedirectUriContainsFragment,
                     nameof(redirectUri));
             }
+
+            // Currenlty only MSAL supports the system browser, on Android and iOS
+            if (usesSystemBrowser &&
+                Constants.DefaultRedirectUri.Equals(redirectUri.AbsoluteUri, StringComparison.OrdinalIgnoreCase))
```

Comment:
Should this instead be called no redirect uri ?

Reply:
What do you mean? The constant ? (this is urn:oob:...)

Reply:
Ah, I see your point, fixing.